ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/task/DeleteJobsTask.groovy

```diff
@@ -17,40 +17,20 @@
 
 package com.marklogic.gradle.task
 
-import com.marklogic.gradle.exception.JobIdsRequiredException
-import com.marklogic.hub.legacy.job.JobDeleteResponse
-import org.gradle.api.tasks.Input
-import org.gradle.api.tasks.Optional
+import com.marklogic.gradle.exception.RetainDurationRequiredException
+import com.marklogic.hub.job.JobDataManager
 import org.gradle.api.tasks.TaskAction
 
 class DeleteJobsTask extends HubTask {
 
-    @Input
-    @Optional
-    public String jobIds
-
-    String getJobIds() {
-        return jobIds
-    }
-
     @TaskAction
-    void deleteJobs() {
-        if (jobIds == null) {
-            jobIds = project.hasProperty("jobIds") ? project.property("jobIds") : null
-        }
-        if (jobIds == null) {
-            throw new JobIdsRequiredException()
+    void cleanJobsData() {
+        def propName = "retainDuration"
+        def retainDuration = project.hasProperty(propName) ? project.property(propName) : null
+        if (retainDuration == null) {
+            throw new RetainDurationRequiredException()
```

Comment:
Just throw a GradleException here with the desired message - there's no value in the indirection from a separate class that won't ever be used except by this task class.